skopt/plots.py

```diff
@@ -230,10 +235,11 @@ def partial_dependence(space, model, i, j=None, sample_points=None,
                 row.append(np.mean(model.predict(rvs_)))
             zi.append(row)
 
-        return xi, yi, zi
+        return xi, yi, np.array(zi).T
 
 
-def plot_objective(result, levels=10, n_points=40, n_samples=250):
+def plot_objective(result, levels=10, n_points=40, n_samples=250,
+                   z_scale='linear'):
```

Comment:
`zscale` for consistency  with `plot_convergence`? (where `yscale` exists, and corresponds to the param name in matplotlib)